Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -430,7 +430,7 @@ public void AddDeclaration(Declaration declaration)
             }
         }
 
-        public void ClearStateCache(VBProject project)
+        public void ClearStateCache(VBProject project, bool setStateChanged = false)
```

Comment:
`setStateChanged` isn't an ideal wording; we're not "setting" anything here, rather *notifying* whoever is listening for parser state events, that the state has changed.

I would have gone with `notifyStateChanged` I think.